static/css/zamboni/zamboni.css

```diff
@@ -2020,6 +2020,7 @@ form .error .note.error {
     -moz-border-radius: 5px;
     -webkit-border-radius: 5px;
     border-radius: 5px;
+    width: 95%;
```

Comment:
I tried 100% width first but the whole blue box was overflowing then. Thats why I set it to 95%. Should I change something else?

Screenshot:-
![ff_req_change](https://cloud.githubusercontent.com/assets/11994135/21566599/67674dc6-ceca-11e6-8a1b-4dbc0d4afd72.png)